exporter/exporterhelper/common.go

```diff
@@ -80,9 +81,9 @@ func WithTimeout(timeoutSettings TimeoutSettings) Option {
 	}
 }
 
-// WithRetry overrides the default RetrySettings for an exporter.
-// The default RetrySettings is to disable retries.
-func WithRetry(config RetrySettings) Option {
```

Comment:
this is a breaking change, should be deprecated

Reply:
Why? `RetrySettings = configretry.BackOffConfig` so the same signature.